deploy/teardown_resources.sh

```diff
@@ -7,31 +7,59 @@ set -euxo pipefail
 set_namespace $CONJUR_NAMESPACE_NAME
 
 configure_cli_pod
+
+# Helm Chart clean-up
+rm -f conjur.pem
+
 if [ "${DEV}" = "false" ]; then
-  $cli_with_timeout "exec $(get_conjur_cli_pod_name) -- conjur variable values add secrets/test_secret \"supersecret\""
+  pushd ../../../secrets-provider/ci
+else
+  pushd ../secrets-provider/ci
+fi
```

Comment:
how about changing this to
```
if [ "${DEV}" = "false" ]; then
  path="../../../secrets-provider/ci"
else
  path="../secrets-provider/ci"
fi

pushd $path
  find . -type f ! -name '*template.yaml' -delete
popd
```

this way the `popd` and `pushd` are aligned. please change `path` to a more meaningful name, i didn't understand what this path stands for.

Also - what is this folder? why do we have a directory named `secrets-provider` under `secrets-provider-for-k8s`?

Reply:
like the suggestion a lot. Implementing

> Also - what is this folder? why do we have a directory named secrets-provider under secrets-provider-for-k8s?
Is actually where the helm chart will sit. We will add a ci folder to our helm chart folder (secrets-provider) for easier testing.

Reply:
can we name it `secrets-provider-helm`? or maybe add a directory so it's `helm -> secrets-provider`? it is not very clear that this is for helm.

Reply:
following how [conjur-oss](https://github.com/cyberark/conjur-oss-helm-chart) placed the chart
conjur-oss-helm-chart (name of repo) > conjur-oss (name of chart)

Reply:
yes but that repo is purely for helm whereas this repo is not. how about my second suggestion? have a directory named `helm` and `secrets-provider` under it?

Reply:
@oburstein-hub what are convension here? If there is a helm chart as part of the project?

Reply:
In the dapng101 follower it was helm/follower. 
According to this it should be helm/secret-provider